server/src/gc.rs

```diff
@@ -158,6 +158,15 @@ async fn run_reap_orphan_chunks(state: &State) -> Result<()> {
     let db = state.database().await?;
     let storage = state.storage().await?;
 
+    let orphan_chunk_id_limit = match db.get_database_backend() {
+        // Default value of --max-allowed-packet https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_max_allowed_packet
+        sea_orm::DatabaseBackend::MySql    => 67108864,
```

Comment:
`max_allowed_packet` sets the binary size of a message to and from mysql.
This is a different thing than the limit in the SQL sense. 
I would just limit that to a value of 1000 instead (I'm not aware of any variable setting the limit, beside the `max_allowed_packet` but this is the size after encoding).
```suggestion
        sea_orm::DatabaseBackend::MySql    => 1000,
```

Reply:
I chose that value according to this stackoverflow answer: https://stackoverflow.com/questions/12666502/mysql-in-clause-max-number-of-arguments.

Reply:
I think the stackoverflow answer says the same thing as what I'm saying.
The limit is a binary size limit, not a limit of arguments.